docs/get-started/Velox.md

```diff
@@ -66,32 +68,34 @@ cd /path_to_gluten
 ./dev/builddeps-veloxbe.sh
 ```
 
-<b>Alternatively you may build gluten step by step as below.</b>
+**Build Velox or Arrow separately**
+
+Scripts under `/path_to_gluten/ep/build-xxx/src` provide `get_xxx.sh` and `build_arrow.sh` to build Velox or Arrow separately, you could use these scripts with custom repo/branch/location.
+
+Velox can use pre-build arrow/parquet lib from ARROW_HOME parsed by --arrow_home to decrease build time.
+Gluten cpp module need a required VELOX_HOME parsed by --velox_home and an optional ARROW_HOME by --arrow_home, if you specify custom ep location, make sure these variables be passed correctly.
 
 ```bash
-## fetch arrow and compile
+## fetch Arrow and compile
 cd /path_to_gluten/ep/build-arrow/src/
+## you could use custom ep location by --arrow_home=custom_path
 ./get_arrow.sh
 ./build_arrow.sh
 
-## fetch velox
+## fetch Velox and compile
 cd /path_to_gluten/ep/build-velox/src/
+## you could use custom ep location by --velox_home=custom_path
 ./get_velox.sh
-## compile velox
+## if you use custom arrow_home, should be specified here by --arrow_home=custom_path
 ./build_velox.sh
```

Comment:
Just a tiny ambiguity issue in the note.
It may be better to also add note to allow/let user specify custom velox_home for `build_velox.sh`. As if user specifies one in `get_velox.sh`, user must have to specify it for `build_velox.sh`. Right?
Same for `build_arrow.sh`. Thanks!

Suggestion:
If you use custom arrow_home or velox_home, they should also be specified here by --arrow_home=custom_path --velox_home=custom_path

Reply:
Already updated, thanks!